### PR TITLE
docs(saas vs oss): add dedicated page

### PR DESCRIPTION
**Pull request by adnanrahic**
This PR adds a dedicated concepts page that explains the feature differences between commercial and oss.

It also includes a few misc fixes and SEO keyword improvements.

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Loom video

Add your loom video here if your work can be visualized

### CODE REVIEW

File: docs/docs/concepts/open-source-vs-commercial-features.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - observability
+  - distributed tracing
+  - testing
+image: https://res.cloudinary.com/djwdcmwdz/image/upload/v1698686403/docs/Blog_Thumbnail_14_rsvkmo.jpg
+---
+
+Tracetest has two distinct feature sets:
+
+- [Hobby Self-hosted Open-source Tracetest Core](https://github.com/kubeshop/tracetest): Free and available on GitHub. Contains the core trace-based testing features. Perfect for hobby usage and non-production workloads.
+- [Commercial Tracetest](https://app.tracetest.io/) (Cloud + On-Prem): The commercial versions of Tracetest include both a cloud-based managed version and an on-prem enterprise version. They contain more advanced features on top of Tracetest Core for larger organizations with platform and onbservability teams.
```
Comment (jfermi) [inline docs/docs/concepts/open-source-vs-commercial-features.mdx:18]:
```suggestion
- [Commercial Tracetest](https://app.tracetest.io/) (Cloud + On-Prem): The commercial versions of Tracetest include both a cloud-based managed version and an on-prem enterprise version. They contain more advanced features on top of Tracetest Core for larger organizations with platform and observability teams.

```

File: docs/docs/examples-tutorials/recipes/running-playwright-performance-tests-with-artillery-and-tracetest.mdx [hunk 2]
```diff
@@ -12,6 +12,12 @@ keywords:
   - artillery
   - load testing
   - performance testing
+  - e2e testing
+  - playwright
+  - end to end testing
+  - end-to-end testing
```
Comment (jfermi) [inline docs/docs/examples-tutorials/recipes/running-playwright-performance-tests-with-artillery-and-tracetest.mdx:18]: This in the list twice?

**Review by jfermi — COMMENTED** [review]
Some editing suggestions.

File: docs/docs/examples-tutorials/recipes/running-playwright-performance-tests-with-artillery-and-tracetest.mdx (hunk 2 above)
Comment (adnanrahic) [reply docs/docs/examples-tutorials/recipes/running-playwright-performance-tests-with-artillery-and-tracetest.mdx:18]: Yeah that's fine. Semrush thinks theyre two distinct keywords.

**Review by danielbdias — APPROVED** [review]

**Pull request merged**